requirements/pip.txt

```diff
@@ -64,6 +64,7 @@ Unipath==1.1
 django-kombu==0.9.4
 mimeparse==0.1.3
 mock==2.0.0
+yamale==1.7.0
```

Comment:
This is assuming yamale will be an application dependency, where I think it's only a testing dependency for now. This should be moved to `testing.txt`